workflow/python/ds-pack/1.1.0-base-python-3.10.13/requirements.txt

```diff
@@ -0,0 +1,11 @@
+boto3==1.26.157
+pandas==1.4.2
+psycopg==3.1.12
+sklearn==0.0
+SQLAlchemy==1.4.5
+sqlalchemy-redshift==0.8.14
+snowflake-sqlalchemy==1.5.0
+peak-sdk==1.0.0
```

Comment:
Should we install `1.1.0`? 🤔

Reply:
Hey i had used `1.1.0` only.  Don't know how it got reverted 🤔 . I've pushed back `1.1.0`.